public/scripts/Table.js

```diff
@@ -38,6 +38,7 @@ var Table = React.createClass({
         return data.map(function(item) {
             var cells = cols.map(function(colData) {
                 return <td> {item[colData.key]} </td>;
+              {item[colData.key]}.sort(sort_by('name1', asec));
```

Comment:
S sir..but im trying to sort entire table based on label Name..once i get the idea of this then i can move to onClick function.without knowing the idea of sorting the entire table , I cant directly get the answers for onClick method